### PR TITLE
You got a fuckin' problem mate? Adds the glass shiv (attempt 2)

### PR DESCRIPTION
**Pull request by LiberatedWaveMan**
## What Does This PR Do

Adds the glass shiv to the game, made with 1 piece of cloth, 1 glass shard, and 5 cable coil; it has a 70% success rate in surgery (compared to the shard's 60% and the kitchen knife's 90%)
## Why It's Good For The Game

I think it would be interesting to have a viable ghetto tool like this, especially since the ramifications aren't immense (you can, after all, find sharp weapons pretty much anywhere); A new option for ghetto surgery should be promising in-game. This _could_ be considered a weapon under space law, since it's purpose-built as one like the spear, but that's not for me to decide.
## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/81937308/a8f84a9a-3519-43bc-8027-b3d053e5e385

https://github.com/ParadiseSS13/Paradise/assets/81937308/6648e02f-617c-40ff-a3ec-c8c9c3f08d9c

glassshiv

## Testing

Abused monkey, did surgery on monkey, killed self, mission successful
## Changelog
:cl:
add: Adds a craftable glass shiv, which can perform surgery or act as a weapon.
/:cl:

### CODE REVIEW

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -58,6 +58,16 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
+/datum/crafting_recipe/shiv
+	name= "Shiv"
```
Comment (Henri215) [inline code/modules/crafting/recipes.dm:62]:
```suggestion
	name = "Shiv"
```

File: code/modules/crafting/recipes.dm [hunk 2]
```diff
@@ -58,6 +58,16 @@
...
+	result = list(/obj/item/kitchen/knife/shiv)
+	reqs = list(/obj/item/stack/cable_coil = 5,
+				/obj/item/shard = 1,
+				/obj/item/stack/sheet/cloth = 1)
+	time = 20
```
Comment (Henri215) [inline code/modules/crafting/recipes.dm:67]:
```suggestion
	time = 2 SECONDS
```

File: code/game/objects/items/weapons/kitchen.dm [hunk 3]
```diff
@@ -138,6 +138,13 @@
 	icon_state = "render"
 	w_class = WEIGHT_CLASS_NORMAL
 
+/obj/item/kitchen/knife/shiv
+	name = "glass shiv"
+	desc = "A haphazardly constructed blade, fashioned from a glass shard wrapped in cloth and tied with wire."
```
Comment (ItsMarmite) [inline code/game/objects/items/weapons/kitchen.dm:143]: This item currently doesn't have a force value, you may want to give it one,.

Comment (LiberatedWaveMan) [reply code/game/objects/items/weapons/kitchen.dm:143]: Intentional, wanted it to inherit the force of a regular knife

Comment (DGamerL) [inline code/game/objects/items/weapons/kitchen.dm:143]: Might just be me but I'd prefer some flavour text over how to make the thing, as I don't need to know that if I already made the damn thing

**Review by DGamerL — COMMENTED** [review]
Can't check the .dmi atm, one small thing

File: code/game/objects/items/weapons/kitchen.dm (hunk 3 above)
Comment (LiberatedWaveMan) [reply code/game/objects/items/weapons/kitchen.dm:143]: Will change this to be more interesting when I’m back at my PC, thanks

**Comment by ATP-Engineer** [comment]
"Are you gonna stick me with that broken piece of plate?"

**Comment by LiberatedWaveMan** [comment]
Flavour text updated to be somewhat funny: "A haphazard sharp object wrapped in cloth. Just like great-great-great-great grandma used to make."

**Review by DGamerL — APPROVED** [review]
items_inhands.dmi my beloathed

**Review by Henri215 — APPROVED** [review]
Items_inhand.dmi 😢

**Comment by Inferno1882** [comment]
Improvised Bladed weapons my beloved. Despite the (relatively) wide selection of blade-like weapons at the disposal of your average Tider, I like this addition.

Can't wait to get arrested 15 times more than usual for making these.

**Comment by S34NW** [comment]
Pardon the random changes, I used your PR as a test case for https://github.com/ParadiseSS13/Paradise/pull/22491

**Review by S34NW — APPROVED** [review]
Needs sprite team approval, someone ping ramon. Otherwise good to go.

**Pull request merged**